stroeer/page/section/v1/section_page.proto

```diff
@@ -61,49 +67,53 @@ message StageConfiguration {
 
 // A single generic item of a stage.
 message StageItem {
-  // Width instructions for rendering the single item.
-  StageItemWidth stage_item_width = 1;
+  // Optional flags/hints for customization only for the stage item.
+  map<string, string> fields = 1;
   // At most one field will be set at the same time.
-  // The delegate of the StageItem can be another Stage, an editorial article or a Commercial.
+  // The delegate of the StageItem can be another teaser stage, an editorial article or a commercial.
   oneof stage_item_type {
-    Stage stage = 2;
+    TeaserStage stage = 2;
     ArticleTeaser article_teaser = 3;
     Commercial commercial = 4;
   }
 }
 
-// Data to render a Commercial as part of a Stage.
-// todo: requirements for this feature are currently missing.
-message Commercial {
+// A non-recursive collection of teaseable items.
+message TeaserStage {
+  // * `stage_layout`: represents a visual layout, which may affect teasers, colors, and other styles.
+  // If entry `stage_layout` is missing, use layout `stream`.
+  // the stage_layout field can be used to configure full page width commercials (e.g. nativendo, t-online empfiehlt)
   map<string, string> fields = 1;
+  repeated TeaserStageItem teaser_stage_items = 2;
+}
+
+// A single teaseable item.
+// alternatives: ClickableItem, LinkedItem, EverythingButAStage
+message TeaserStageItem {
```

Comment:
I think the name "Teaser" implies that only an Article Teaser can be used here, I'd prefer "SubStage" naming or something like "ChildStage" or "StageChildItem"

Reply:
I know what you mean, but doesn't a Commercial tease a product?

Reply:
It does, but this is not our Business Domain. If a user clicks on a Teaser, he is not using our Page anymore. A "Teaser" only teases stuff from our domain, in my opinion.

Reply:
// alternatives: ClickableItem, LinkedItem, EverythingButAStage